appconfig/appconfig.go

```diff
@@ -17,13 +19,15 @@ type AppConfig struct {
 	UaResolver  useragent.Resolver
 
 	AuthorizationService *authorization.Service
+	QueryLogsWriter      *io.Writer
```

Comment:
io.Writer interface -> it's already a pointer

Reply:
fixed